test/test.js

```diff
@@ -584,20 +586,61 @@ exercise.forEach(function(exerciseMode) {
 
     // https://fetch.spec.whatwg.org/#response-class
     suite('Response', function() {
-      test('default status is 200 OK', function() {
+      featureDependent(test, emptyDefaultStatusText, "default status is 200 ''", function() {
         var res = new Response()
         assert.equal(res.status, 200)
-        assert.equal(res.statusText, 'OK')
+        assert.equal(res.statusText, '')
         assert.isTrue(res.ok)
       })
 
-      test('default status is 200 OK when an explicit undefined status code is passed', function() {
-        var res = new Response('', {status: undefined})
+      featureDependent(
+        test,
+        emptyDefaultStatusText,
+        "default status is 200 '' when an explicit undefined status code is passed",
+        function() {
+          var res = new Response('', {status: undefined})
+          assert.equal(res.status, 200)
+          assert.equal(res.statusText, '')
+          assert.isTrue(res.ok)
+        }
+      )
+      featureDependent(test, !emptyDefaultStatusText, 'default status is 200 OK', function() {
+        var res = new Response()
         assert.equal(res.status, 200)
         assert.equal(res.statusText, 'OK')
         assert.isTrue(res.ok)
       })
 
+      featureDependent(
+        test,
+        !emptyDefaultStatusText,
+        'default status is 200 OK when an explicit undefined status code is passed',
+        function() {
+          var res = new Response('', {status: undefined})
+          assert.equal(res.status, 200)
+          assert.equal(res.statusText, 'OK')
+          assert.isTrue(res.ok)
+        }
+      )
+      featureDependent(test, exerciseMode !== 'native', "default status is 200 ''", function() {
+        var res = new Response()
+        assert.equal(res.status, 200)
+        assert.equal(res.statusText, '')
+        assert.isTrue(res.ok)
+      })
+
+      featureDependent(
+        test,
+        exerciseMode !== 'native',
```

Comment:
These two `exerciseMode !== native` tests both seem redundant and they can be removed. The `emptyDefaultStatusText` condition already covers the `exerciseMode !== native` case